src/app/services/xml-parsers/structure-xml-parser.service.ts

```diff
@@ -24,7 +24,7 @@ export class StructureXmlParserService {
     const front: XMLElement = el.querySelector(this.frontTagName);
     const body: XMLElement = el.querySelector(this.bodyTagName);
 
-    const pbs = Array.from(el.querySelectorAll(this.pageTagName));
+    const pbs = Array.from(el.querySelectorAll(this.pageTagName)).filter((p) => !p.getAttribute('ed'));
```

Comment:
Is this use of the attribute documented somewhere?

Reply:
Yes, the TEI guidelines referring to <pb> say that:

> The more specialized attributes break, ed, or edRef should be preferred when the intent is to indicate whether or not the page break is word-breaking, or to note the source from which it derives.

https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-pb.html

Reply:
The description do not refer to "filtering out" pbs that have `ed` attribute though.
I am not sure this is the right thing to do